### DISCUSSION TITLE
Can you tell me how to implement password recovery for flutter ?

### DISCUSSION THREAD

**Discussion by AIQMEDIA** [supabase/supabase#1989] [Questions]
I am using below code :

final response = await supabaseClient.auth.api.resetPasswordForEmail(email);

but after I click the password recovery link sent to me in. mail, I go to a blank page with address localhost:3000/#. 

Please help, what site URL should I use to redirect it to my app and update password correctly ?

**Comment by phamhieu** [comment]
Hi @AIQMEDIA, you can set `redirect_to` param on **resetPasswordForEmail** method with the latest version of [supabase-dart](https://pub.dev/packages/supabase). 

```
client.auth.api.resetPasswordForEmail(
  'email', 
  options:AuthOptions(redirectTo: 'io.supabase.demoapp://recovery-callback')
);
```

Remember to add your redirect url to `Additional Redirect URLs` field  under Supabase Authentication settings page. 
<img width="842" alt="Screenshot 2021-06-28 at 8 18 55 AM" src="https://user-images.githubusercontent.com/689843/123566293-8abe2100-d7e9-11eb-9f69-9bde647c427a.png">
